suites/reef/rgw/tier-2_ssl_rgw_ecpool_test.yaml

```diff
@@ -90,7 +90,7 @@ tests:
                     nodes:
                       - node7
                   spec:
-                    ssl: false
+                    ssl: true
```

Comment:
Can you please also update the other ssl suite with this value
https://github.com/red-hat-storage/cephci/blob/master/suites/quincy/rgw/tier-2_rgw_ssl_s3tests.yaml

Reply:
Done , wonder why these suites had ssl disabled.

Reply:
looks like it was changed temporarily in this pr: https://github.com/red-hat-storage/cephci/commit/638a0c357e03574e03785b97e56c248c67240677